dhqa_data.py

```diff
@@ -58,8 +59,11 @@ def get_post_info(div, topic_url, feed):
     [comment.extract() for comment in threadpost.findAll(
         text=lambda text:isinstance(text, Comment))]
 
-    # get post content
-    info['content'] = threadpost.div.prettify()
+    # get post html content
+    info['html_content'] = threadpost.div.prettify()
+
+    # extract text from post on html page
+    info['content'] = threadpost.find('div', class_='post').get_text()
```

Comment:
Seems a little odd to me we're not accessing the div the same way for both of these; unlikely but possible we could get different content. I think the post is always the first div, but does seem more reliable to look for it by class to make sure. Maybe find and assign to a variable first and then use for both html and text content?

Reply:
Ya I agree that there might be issues doing this two different ways. The reason I had to use `post` was because `threadpost` contains a number of paragraph tags and if I just call `get_text` on it, soup doesn't remove the html tags. I was wondering why we were keeping the html tags in the first place? Do you think people would be interested in them? I'm not entirely sure how to reformat this given the issues with soup, but let me know if you have thoughts!

Reply:
Ah, interesting. 

My use case for the html is to look at the sites people were linking to in their comments.   I haven't checked what get text returns, but I expect it would omit attributes like that. 

I recommend you find the post div the way you need to for the text content and then use that to generate both html and text output.

Reply:
So I went back and looked at threadpost and all I had to do was call `get_text` on the div so made that change to standardize how we're getting the content. Also like the ideas of looking for replies and links!